ui/k-info-panel/show_circuit.kytos

```diff
@@ -1044,6 +1055,16 @@ module.exports = {
     clear: both;
     justify-content: space-between;
   }
+  .left-bottom-buttons {
+    align-self: flex-end;
+  }
+  .right-bottom-buttons {
+    border: 2px solid #1e1d1d;
```

Comment:
This is a square that is around the check box to avoid `s_vlan` and the button to redeploy. It could be removed or change the color. See picture above

Reply:
I liked it, it helped to understand they are related. 

```
src/assets/styles/variables.scss
8:$kytos-gray: #515151;
```

![20241212_090211](https://github.com/user-attachments/assets/d1bbfeca-b728-4680-a602-b7752457f7e5)

Can it be kytos-gray #515151? Cosmetic stuff but looks like it blends well. See if you like it too, if positive let's go for gray otherwise as it is it's fine too.

Reply:
A'right, changed to `#515151`